e2e/lvm/pvc_test.go

```diff
@@ -15,75 +16,198 @@ import (
 const (
 	timeout  = time.Minute * 5
 	interval = time.Second * 30
+	size     = "5Gi"
 )
 
-// Test to verify the PVC status
 func PVCTest() {
 
 	Describe("PVC Tests", func() {
-		var filepvc *k8sv1.PersistentVolumeClaim
-		var filepod *k8sv1.Pod
-		var blockpvc *k8sv1.PersistentVolumeClaim
-		var blockpod *k8sv1.Pod
+		var pvc *k8sv1.PersistentVolumeClaim
+		var pod *k8sv1.Pod
+		var snapshot *snapapi.VolumeSnapshot
+		var clonePvc *k8sv1.PersistentVolumeClaim
+		var clonePod *k8sv1.Pod
+		var restorePvc *k8sv1.PersistentVolumeClaim
+		var restorePod *k8sv1.Pod
+		test := tests.DeployManagerObj.GetCrClient()
+
+		Context("create pvc, pod, snapshots, clones", func() {
+			It("Tests PVC operations for VolumeMode=file system", func() {
+				By("Creating pvc and pod")
+				pvc = tests.GetSamplePVC(tests.StorageClass, size, "lvmfilepvc", k8sv1.PersistentVolumeFilesystem)
+				pod = tests.GetSamplePod("lvmfilepod", pvc.Name)
+				err := test.Create(context.TODO(), pvc)
```

Comment:
Are these resources cleaned up in case of error?

Reply:
In the end, we are deleting this namespace only, so anyway all of these will get removed.